app/managers.py

```diff
@@ -0,0 +1,43 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema")
```

Comment:
Please, add extension to name of your file

Reply:
Done.